config.yaml

```diff
@@ -24,6 +27,9 @@
 # # Allow network configuration via indexer manifests.
 # indexer_net_config: false
 
+# # Maximum amount of time (seconds) that an indexer's `handle_events` function can take before being timed out.
```

Comment:
Perhaps we can change this to something different as an indexer operator may not know that `handle_events()` is what _actually_ runs under the hood. Maybe something like
> Maximum length of time (in seconds) that an indexer's event handler can run before timing out.